tailwind.config.js

```diff
@@ -12,7 +12,15 @@ module.exports = {
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
+      colors: {
+        customBlue: '#E6F2FF',
+        textBlue_1 : '#0080FF',
```

Comment:
Rename `textBlue_1` to `textBlue` and update all relevant instances where it is used.

Reply:
BREAKING CHANGES : Resolved!